include/crm/common/internal.h

```diff
@@ -162,13 +162,6 @@ int pcmk__compress(const char *data, unsigned int length, unsigned int max,
 // Example: crm_info("Found %d node%s", nnodes, pcmk__plural_s(nnodes));
 #define pcmk__plural_s(i) pcmk__plural_alt(i, "", "s")
 
-static inline char *
-crm_concat(const char *prefix, const char *suffix, char join)
-{
-    CRM_ASSERT(prefix && suffix);
-    return crm_strdup_printf("%s%c%s", prefix, join, suffix);
-}
-
```

Comment:
This patch makes things so much easier to read.  The argument order to crm_concat was really awkward.

Reply:
Yeah that was my favorite too :)